### PR TITLE
Add swagger data models

### PR DESCRIPTION
**Pull request by Dracco30**

### CODE REVIEW

**Comment by studentgrad0419** [comment]
Hi @Dracco30 do you have any updates for this?

**Comment by Dracco30** [comment]
> Hi @Dracco30 do you have any updates for this?

I think I have made the right change this time, however I noticed that the branch was changed by others, should I merge their code into mine?

**Comment by Dracco30** [comment]
@studentgrad0419 @rahilkadakia Hi, the automatic checks have been passed, please check this.

File: service/routes.py [hunk 1]
```diff
@@ -5,11 +5,93 @@
...
+
+create_wishlist_model = api.model(
+    "Wishlist",
+    {
+        "name": fields.String(required=True, description="The Name of the wishlist"),
+        "owner_id": fields.Integer(
+            required=True, description="The owner id of the wishlist"
+        ),
+    },
+)
```
Comment (studentgrad0419) [inline service/routes.py:56]: I think we now also have a "is Public:" field as well

**Review by studentgrad0419 — COMMENTED** [review]
Hi @Dracco30 I think for defining Swagger models this looks good. There is a missing attribute for wishlist "Is Public" that should be added.

outside of this I think it can be merged

File: service/routes.py (hunk 1 above)
Comment (Dracco30) [reply service/routes.py:56]: Hi @studentgrad0419 , I think the 'public function' is already in my code, 
image

Comment (Dracco30) [reply service/routes.py]: image
I think it's ready to merge now.

**Comment by jay-goettelmann-nyu** [comment]
@Dracco30 I'm closing this PR because it has a bunch of extra commits that are not related and that is a little confusing.  I create a new PR for these changes.  https://github.com/CSCI-GA-2820-FA23-003/wishlists/pull/127

**Pull request closed without merging**